### PR TITLE
Add SBUS2 receiver type

### PR DESCRIPTION
**Pull request by mmosca**
Companion to rotorflight/rotorflight-firmware#130

Adds SBUS2 option to receiver tab.

### CODE REVIEW

**Comment by mmosca** [comment]
Unfortunatelly, I can't build the configurator on my Mac (Apple Silicon Sonoma 14.5), so some testing help is welcome. Firmware was validated by setting serialrx_provider to SBUS2 and enabling telemetry.

**Comment by Oats87** [comment]
> Unfortunatelly, I can't build the configurator on my Mac (Apple Silicon Sonoma 14.5), so some testing help is welcome. Firmware was validated by setting serialrx_provider to SBUS2 and enabling telemetry.

What's your issue building? I can build on my Sonoma Mac, but it took a bit of finagling to get the dependencies just right.

The steps here worked for me: https://github.com/rotorflight/rotorflight-configurator/blob/master/HOWTO.md#setup only had to specify `nvm install 16`

**Comment by mmosca** [comment]
I haven't tried to dig deeper because I know nws.js had lots of issues supporting apple silicon macs (works on intel ones just fine).

I will get some build logs later. nvm install 16 may do the trick by forcing it to be treated as an intel mac, My node version is probably newer and has support for apple silicon.

**Comment by rotorflight** [comment]
It won't work with any other Node version than 16.5.
nvm should install the right version, which is defined in .nvmrc

File: src/js/tabs/receiver.js [hunk 1]
```diff
@@ -90,6 +90,7 @@ TABS.receiver = {
         { name: 'None',                 id: 0,   feature: null,           telemetry: 0,           visible: true, },
         { name: 'TBS CRSF',             id: 9,   feature: 'RX_SERIAL',    telemetry: 0x0010378F,  visible: true, },
         { name: 'Futaba S.BUS',         id: 2,   feature: 'RX_SERIAL',    telemetry: 0,           visible: true, },
+        { name: 'Futaba S.BUS2',        id: 15,  feature: 'RX_SERIAL',    telemetry: 0xFFFFFFFF,  visible: true, },
```
Comment (rotorflight) [inline src/js/tabs/receiver.js:93]: It looks like the telemetry sensors are not configurable, right?
If so, the configurable sensor field should be `telemetry: 0`

Comment (mmosca) [reply src/js/tabs/receiver.js]: Changed to 0.

**Comment by mmosca** [comment]
For the record, the changes in master helped, but unfortunatelly, it still does not build on apple silicon mac.

```
node:internal/modules/cjs/loader:1614
  return process.dlopen(module, path.toNamespacedPath(filename));
                 ^

Error: The module '/Users/mosca/dev/rotorflight-configurator/node_modules/libxmljs/build/Release/xmljs.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 120. This version of Node.js requires
NODE_MODULE_VERSION 127. Please try re-compiling or re-installing
```

Based on the error, it is probably a random dependency without apple silicon support.

I managed to work around it by forcing my shell to run into x86 mode and doing a git clean to remove apple silicon modules.

To switch to intel cpu emulation run:
```
arch -x86_64 zsh
```

After that you can follow normal procedures to build the configurator.

**Review by rotorflight — APPROVED** [review]

**Pull request merged**